pkg/manager/impl/validation/project_validator.go

```diff
@@ -25,7 +25,11 @@ func ValidateProjectRegisterRequest(request admin.ProjectRegisterRequest) error
 	if request.Project == nil {
 		return shared.GetMissingArgumentError(shared.Project)
 	}
-	return ValidateProject(*request.Project)
+	project := *request.Project
+	if err := ValidateEmptyStringField(project.Name, projectName); err != nil {
+		return err
+	}
+	return ValidateProject(project)
```

Comment:
do you want to move more of the validation (e.g. description, labels, etc) out of ValidateProject and into this method?

Reply:
oh perhaps labels is unnecessary, but description maybe

Reply:
We are not validating emptiness of description in `VadlidateProject`. Do you mean we want to enforce non-empty for description as well?

Reply:
ah you're right again, sorry skimmed over that

Reply:
No worries. :)